### DISCUSSION TITLE
How to get alt text of images in firehose?

### DISCUSSION THREAD

**Discussion by git-scott** [MarshalX/atproto#285] [Questions] [answered]
Hi, sorry for the novice question but I’m struggling to work out how to access alt-text for embedded images in posts.  How is this done, assuming there may be multiple images’ text to parse per post.

Thanks

**Comment by MarshalX** [comment, marked as answer]
Hi, in `data_filter.py` [line no 22](https://github.com/MarshalX/bluesky-feed-generator/blob/7876ad5cf617e29aac9e481e515fc6ac627e3a97/server/data_filter.py#L22) you can add this one:

```python
if post_with_images:
    for image in record.embed.images:
        print('Alt:', image.alt)
```

**Comment by git-scott** [comment]
Thanks very much!
